tests/upgradePMM_test.js

```diff
@@ -105,6 +102,19 @@ Scenario(
   },
 );
 
+Scenario(
+  'PMM-T577 Verify user is able to see IA alerts before upgrade @pre-upgrade @ami-upgrade @pmm-upgrade',
```

Comment:
How would this work for older versions of PMM? when we didn't implement this feature.

Reply:
Good point, fixing it